README.md

```diff
@@ -543,6 +558,9 @@ executing the request and in the same order received.
 On devices that support it, this method resets the system the same way pressing
 the RESET button/slider on the SNES hardware console does.
 
+#### [ResetToMenu](https://github.com/alttpo/sni/blob/main/protos/sni/sni.proto#L21)
```

Comment:
i'm not entirely sure what line this is supposed to point at - it seems like all the other line links are to random places (presumably because the proto file changes faster than the docs), so I picked the rpc definition.

Reply:
Ah good catch. Yeah the sni.proto file grew over time so the specific line number links in the README are no longer valid. Hm don't have a good way to combat this honestly other than trying to stay on top of it and validating the links from README.